### DISCUSSION TITLE
Can't open the window

### DISCUSSION THREAD

**Discussion by sankaviv1** [kangyu-california/PersistentWindows#205] [Q&A] [answered]
Hi, I can't figure out how to open the dialog. I have this icon in the taskbar 
![image](https://user-images.githubusercontent.com/95748224/217847303-ad360cd7-adf3-4257-9b75-681f9451d270.png)

and these are the right click options
![image](https://user-images.githubusercontent.com/95748224/217847399-7f08a6d5-e747-4979-8ccb-714b734b41b8.png)

double clicking just captures a screenshot. how do i open the window so i can set the properties etc TIA

**Comment by kangyu-california** [comment, marked as answer]
There is no classic "main" window for tiny application like PW.
Most of the static settings are applied using [command line](https://github.com/kangyu-california/PersistentWindows/blob/master/Help.md), other settings can be turned on/off in the right click options.
